pkg/valid/validations.go

```diff
@@ -26,12 +26,14 @@ const (
 	AppNameRegExp  = `\A[a-z0-9]+(?:-[a-z0-9]+)*\z`
 	TeamNameRegExp = AppNameRegExp
 	EnvNameRegExp  = AppNameRegExp
+	CommitIDRegExp = `^[0-9a-f]{40}$`
```

Comment:
Commit ids could be in upper-case form, so the regular expression should support that as well.
```suggestion
	CommitIDRegExp = `^[0-9a-fA-F]{40}$`
```